docker-compose.yml

```diff
@@ -38,6 +38,20 @@ services:
     env_file: .env
     container_name: mysql
 
+  postgres:
+    image: postgres:9.6.3
+    environment:
+      - POSTGRES_DB=${DB_NAME}
+      - POSTGRES_USER=${DB_USER}
+      - POSTGRES_PASSWORD=${DB_PASSWORD}
+    volumes:
+      - ./docker:/app/docker
+      - ./docker/psql:/etc/psql/conf.d
```

Comment:
Is this necessary any more? we don't have a `psql/conf.d`? Can maybe leave it here in case we do want one later? Or put a blank placeholder file there so it's less confusing?